src/config.rs

```diff
@@ -204,6 +226,22 @@ fn parse_optional_size(val: &str) -> Result<Option<u64>> {
     })
 }
 
+fn verify_mount_point(val: &str) -> Result<PathBuf> {
+    let path = PathBuf::from(val);
+
+    if path.is_relative() {
+        return Err(anyhow!("mount-point {} is not absolute", val));
+    }
+
+    for component in path.components() {
+        if let std::path::Component::ParentDir = component {
```

Comment:
No other bit uses fully-qualified identifiers, a `use` clause (potentially with a rename` would be more consistent.

Also, isn't this whole thing just `path.components().any(|c| c == Component::ParentDir)`?

Reply:
Ack.